integ/large_data_attributes_test.go

```diff
@@ -0,0 +1,119 @@
+package integ
+
+import (
+	"context"
+	"fmt"
+	"github.com/indeedeng/iwf/gen/iwfidl"
+	"github.com/indeedeng/iwf/integ/workflow/signal"
+	"github.com/indeedeng/iwf/service"
+	"github.com/indeedeng/iwf/service/common/ptr"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+)
+
+func TestLargeDataAttributesTemporalContinueAsNew(t *testing.T) {
+	if !*temporalIntegTest {
+		t.Skip()
+	}
+	for i := 0; i < *repeatIntegTest; i++ {
+		doTestLargeQueryAttributes(t, service.BackendTypeTemporal, minimumContinueAsNewConfigV0())
+		smallWaitForFastTest()
+	}
+}
+
+func doTestLargeQueryAttributes(t *testing.T, backendType service.BackendType, config *iwfidl.WorkflowConfig) {
+	if !*temporalIntegTest {
+		t.Skip()
+	}
+	assertions := assert.New(t)
+
+	// start test workflow server
+	wfHandler := signal.NewHandler()
```

Comment:
Nit: Wouldn't it be better to create a handler class for this specific test?

Reply:
I feel like using the existing is better. The test only need a workflow to do nothing, so that we can set some large DAs. The existing signal workflow is exactly like that